config/runtime.exs

```diff
@@ -56,6 +56,9 @@ if config_env() in [:prod, :demo] do
     ],
     secret_key_base: secret_key_base
 
+  config :wanda,
+    oas_server_url: System.get_env("OAS_SERVER_URL")
```

Comment:
Comment: As you said, this addition could have implications in the [default RPM config](https://github.com/trento-project/wanda/blob/main/packaging/suse/rpm/systemd/trento-wanda.example), the [helm chart](https://github.com/trento-project/helm-charts/blob/main/charts/trento-server/charts/trento-wanda/templates/configmap.yaml) and the [docs](https://github.com/trento-project/docs/blob/main/trento/adoc/trento-systemd-install.adoc).